packages/app-project/package.json

```diff
@@ -36,7 +36,8 @@
     "react-pose": "~4.0.2",
     "react-progressive-image": "~0.6.0",
     "styled-components": "~4.1.1",
-    "url-parse": "~1.4.4"
+    "url-parse": "~1.4.4",
+    "validator": "^10.10.0"
```

Comment:
Can we add the package using tilde? I couldn't figure out how to default it to that using lerna and npm 😬